src/models/People.js

```diff
@@ -78,7 +78,19 @@ static async displayPeopleProfessional() { // returns array of Timesheet Table
         {name: 'people_id', value:{longValue: userId}},
       ]);
 
-      const capabilities = rawCapabilities.records.map(([{ stringValue: capability }])=> capability);
+      const capabilities = rawCapabilities.records.reduce((acc, curr) => {
+        const [{ stringValue: capability },
+                { stringValue: params1 },
+                { stringValue: params2 },
+                { stringValue: params3 }] = curr;
+
+        return [...acc, 
+                capability, 
```

Comment:
`capability` should not be given outright. `capability` should be given only if params1 is NULL AND params2 is NULL AND params3 is NULL.

This makes it possible to have `capability(Pedro)` without `capability`.